includes/D3D/D3DDebugLogger.hpp

```diff
@@ -13,14 +13,12 @@ class D3DDebugLogger {
 	D3DDebugLogger(const Args& arguments);
 	~D3DDebugLogger() noexcept;
 
-#ifdef __ID3D12InfoQueue1_INTERFACE_DEFINED__
 	void UnregisterCallBack() const noexcept;
 
 	static void MessageCallback(
 		D3D12_MESSAGE_CATEGORY category, D3D12_MESSAGE_SEVERITY severity,
 		D3D12_MESSAGE_ID id, LPCSTR pDescription, void* pContext
 	);
-#endif
 private:
 #ifdef __ID3D12InfoQueue1_INTERFACE_DEFINED__
 	DWORD m_callBackCookie;

```

Comment:
No need to put this one under the macro. Then you didn't have to touch its initialisation in the ctor.

Reply:
done